src/ServiceControl/MessageFailures/Api/GetErrorById.cs

```diff
@@ -54,6 +54,7 @@ private static FailedMessageView Map(FailedMessage message, IAsyncDocumentSessio
 
             var metadata = processingAttempt.MessageMetadata;
             var failureDetails = processingAttempt.FailureDetails;
+            var wasEdited = message.ProcessingAttempts.Last().Headers.ContainsKey("ServiceControl.EditOf");
```

Comment:
no `const` somewhere?

Reply:
It appears that there are no constants for `ServiceControl.` prefixed headers anywhere in the codebase. Probably something that should be cleaned up. Not sure if it should be done as part of this work.